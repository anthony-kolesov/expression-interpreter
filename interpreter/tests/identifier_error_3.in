# Reference to unknown variable between numbers.
out 4 + xyz + 4
