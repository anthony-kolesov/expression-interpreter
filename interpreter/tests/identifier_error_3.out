ERROR:2:Unknown identifier: xyz
